src/About/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './style.scss';
+import divider from '../image/Divider.png';
+import cookImg from '../image/cook.png';
+import dish from '../image/dish.png';
+
+function About() {
+  return (
+    <section id="about" className="wrapper">
```

Comment:
class wrapper is to general. do u use it on the other components?

Reply:
i have changed the name of wrapper